### ISSUE TITLE
Tag 4.2.0 has gone missing.

### ISSUE THREAD

**Issue by peterwilsoncc** [stuttter/ludicrousdb#118]
d5d27fc4e19ccfeecac627130e1ec68a800ca381 was previously tagged as `4.2.0` but this appears to have gone missing on or about September 26 according to when my pipeline started failing for a site using https://github.com/humanmade/altis-cloud/commit/cb1b83ad4c806e94bf445f206fbb8b130b3dfb45 (v3.0.15).

Are you able to take a look and retag?

**Comment by peterwilsoncc** [comment]
Edit: typo fixed 4.2.0 rather than 4.2.2. Sorry.

**Comment by JJJ** [comment]
@spacedmonkey fixed this with `git push --tags  -f`.

Truly not sure why this happened. Sorry for it, though. And I'll keep an eye on it.

**Comment by peterwilsoncc** [comment]
Thanks for the prompt fix folks. `composer install` is much happier now.

**Comment by rmccue** [comment]
This happened again, so we've created a mirror on Packagist: https://packagist.org/packages/humanmade/ludicrousdb

**Comment by JJJ** [comment]
I truly have no idea WTF is happening.

Packagist is our only webhook, and there aren’t any GitHub integrations.

There’s no way to audit this activity on GitHub, that I can see.

I’ve re-published the Tag/Release. Maybe I’ll reach out to GitHub support and see if they can help.

**Comment by JJJ** [comment]
Ticket #930039 opened with GitHub support.

Sorry, again. I’ll do everything I can to prevent this from happening so you don’t need to maintain a mirror in perpetuity.

**Comment by JJJ** [comment]
GitHub support helped me figure this out.

I'd setup a mirror on the jjj.software GitLab instance, and that mirror was missing the 4.2.0 and 4.0.0 tags. When I pushed to the mirror, it would push back to GitHub with the `--mirror` flag set, causing those tags to eventually be removed.

GitHub, the website/service, doesn't outright delete a release when a tag goes missing, it simply sets it to draft status.

I've deleted my mirror to avoid anything like this ever happening again. If I need to mirror in the future, I'll be more careful.

@rmccue y'all can remove your mirror if you'd like. Sorry, again, everyone.
